assemble/src/main/resources/LICENSE

```diff
@@ -1,982 +0,0 @@
-
```

Comment:
If we don't bundle anything extra in the -bin.tar.gz, we don't need a special LICENSE file. We can use the same one for the source release.